src/apps/stunclient/stunclient.c

```diff
@@ -28,12 +28,18 @@
  * SUCH DAMAGE.
  */
 
-#include <err.h>
 #include <stdlib.h>
 #include <stdio.h>
 #include <string.h>
 #include <time.h>
-#include <unistd.h>
+#if defined(_MSC_VER)
+    #include <getopt.h>
+#else
+    #include <unistd.h>
+    #if !defined(WINDOWS)
+        #include <err.h>
```

Comment:
is this include required here?
mainuclient.c does not include it (rest of change is very similar)

Reply:
Of course it does, please understand it carefully.